rules/cross-platform/execution_python_script_in_cmdline.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2021/01/13"
+maturity = "production"
+updated_date = "2021/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a Python script is executed from the commandline input. Attackers often use this method to execute
+malicious scripts and avoiding writing it to disk.
+"""
+false_positives = ["Legitimate Python scripting activity."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Python Script Execution via the CommandLine"
```

Comment:
```suggestion
name = "Python Script Execution via Command Line"
```

Reply:
took out suspicious because its not necessarily suspicious, but could be worth looking into depending on the org